### DISCUSSION TITLE
Tabs that hold modules?

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#1887] [Ideas] [answered]
I'm a fan of Tabs.  When composing various tabs it would be nice to pull in different modules.  I wonder if you could put a container in a tab and then load a module into it?

**Comment by sbwalker** [comment, marked as answer]
How about create a Theme with a custom menu which renders a list of Pages visually as a tab strip. Then when you click on a tab the framework will load the Modules which were defined for that Page.

**Reply by markdav-is** [reply]
Thanks for chiming in.  That's interesting.  I'll let you know if I can pull it off.
